client/src/main/python/slipstream/executors/MachineExecutor.py

```diff
@@ -81,9 +81,12 @@ def _execute_state(self, state):
         except KeyboardInterrupt:
             raise
         except (SystemExit, Exception) as ex:
-            util.printError('Error executing node, with detail: %s' % ex)
-            traceback.print_exc()
-            self._fail(ex)
+            if isinstance(ex, SystemExit) and str(ex).startswith('Terminating on signal'):
```

Comment:
Is the language always set to English?  Or will this fail is the machine is running by default in another locale?